<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoWriteDialog</class>
 <widget class="QDialog" name="InfoWriteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="CommonPushButton" name="pushButtonInfoWriteReturn">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>508</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoTitle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string>主题</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonInfoWriteSend">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>427</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditReceiverName">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>215</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
</string>
   </property>
   <property name="text">
    <string>接收者</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MyLineEdit" name="lineEditInfoTitle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>主题</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="framePanel">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>17</y>
     <width>700</width>
     <height>570</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameInfoBox">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>97</y>
     <width>662</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: 2px solid #C0C0C0;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoReceiver">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>接收者</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>150</y>
     <width>662</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #C0C0C0;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>203</y>
     <width>662</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #C0C0C0;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>20</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #C0C0C0;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoBody">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>信息内容:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="MyTextEdit" name="textEditInfoBody">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>273</y>
     <width>550</width>
     <height>260</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #C0C0C0;</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>,white</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>信息发送</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonInfoWriteAddress">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>346</y>
     <width>230</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>通信录</string>
   </property>
  </widget>
  <zorder>framePanel</zorder>
  <zorder>frameInfoBox</zorder>
  <zorder>pushButtonInfoWriteReturn</zorder>
  <zorder>labelInfoTitle</zorder>
  <zorder>pushButtonInfoWriteSend</zorder>
  <zorder>lineEditReceiverName</zorder>
  <zorder>lineEditInfoTitle</zorder>
  <zorder>labelInfoReceiver</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line</zorder>
  <zorder>labelInfoBody</zorder>
  <zorder>textEditInfoBody</zorder>
  <zorder>label_title</zorder>
  <zorder>pushButtonInfoWriteAddress</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyTextEdit</class>
   <extends>QTextEdit</extends>
   <header>mytextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditReceiverName</tabstop>
  <tabstop>lineEditInfoTitle</tabstop>
  <tabstop>pushButtonInfoWriteSend</tabstop>
  <tabstop>pushButtonInfoWriteReturn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
